### ISSUE TITLE
Pagination should use <va-pagination> component

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12903]
## Description
The pagination for the location search results should be added using the `<va-pagination>` [component](https://design.va.gov/components/pagination). This will ensure that the element has the proper HTML markup and ARIA, especially the navigation role which will help screen readers users find and quickly access this section of the page.

## Acceptance Criteria
- [ ] Ensure pagination uses the `<va-pagination>` component
- [ ] Accessibility lead review

**Comment by laflannery** [comment]
This was done when the react component was updated to the web component
